css/styles.css

```diff
@@ -0,0 +1,234 @@
+/* pasek nawigacji------------ */
+nav .logo {
```

Comment:
Try not to use the html selectors. It's better to add class to the `nav` tag e.q. `<nav class="first-page-navigation">` and select here by using `first-page-navigation . logo`.  This is because hanig a bigger project you clould have many nav tags that could match your selector and have wrong styles. Also try to be more precise with the classes names e.q instead of `logo` chnage it to `navigation-brand-logo` that the class is self explainatory.